src/content/docs/en/reference/experimental-flags/static-import-meta-env.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Experimental static import.meta.env
+sidebar:
+  label: Static import.meta.env
```

Comment:
```suggestion
title: Experimental environment variable inlining
sidebar:
  label: Environment variable inlining
```
@florian-lefebvre - Chris and I were talking about having inline code in a page title, and how all the other features were descriptions vs. the name of the flag and we tried to think of a way to *explain* the feature with a descriptive title.

Chris's suggestion was better than my `Experimental not touching your environment variables anymore`! Do you think this accurately describes the feature/content of the page? Is there anything else you'd suggest?